src/main/scala/cromwell/engine/db/slick/SymbolComponent.scala

```diff
@@ -93,4 +96,12 @@ trait SymbolComponent {
       workflowExecution <- symbol.workflowExecution
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield symbol)
+
+  val symbolsForWorkflowOutput = Compiled(
+    (workflowExecutionUuid: Rep[String]) => for {
+      symbol <- symbols
+      if symbol.index === -1 && symbol.reportableResult === true
```

Comment:
I agree, so is it even possible for one to be in the database in the first place?  i.e. do you need to test for `index === -1`?